components/builder-depot/src/server.rs

```diff
@@ -2602,6 +2946,16 @@ where
     r
 }
 
+fn depot_client_from_config(config: &Config) -> Option<DepotClient> {
+    if let Some(ref upstream_depot) = config.upstream_depot {
+        // TODO: we keep running into this issue where PRODUCT and VERSION aren't available and
+        // hardcoding values. how do we avoid this?
+        Some(DepotClient::new(upstream_depot, "hab", "0.55.0", None).unwrap())
```

Comment:
The product should probably not be "hab" .. maybe "depot", and leave version at 0.0.0 until we can retrieve the build version?

Reply:
Good call